packages/mutation-testing-elements/src/components/mutation-test-report-popup/mutation-test-report-popup.scss

```diff
@@ -18,18 +18,20 @@
 }
 
 /* Popup arrow */
-.popover::before {
+.popover-arrow {
     content: "";
     position: absolute;
     top: -10px;
     left: 50%;
-    margin-left: -5px;
-    border-width: 5px;
+    margin-left: -6px;
+    border-width: 6px;
     border-style: solid;
-    border-color: transparent transparent #555 transparent;
+    border-left-color: transparent !important;
```

Comment:
The border-color is set by the bootstrap class (see below comment), which makes the whole element that colour. To override it and only make the arrow (border-bottom) the colour I used `!important`. I'm open to suggestions on other ways to do it, because it does feel a bit hacky (but most things I do in CSS do)

Reply:
Fixed with my new implementation 👍